cristin-person/src/test/java/no/unit/nva/cristin/person/handler/person_api_methods.feature

```diff
@@ -0,0 +1,54 @@
+Feature: API tests for Cristin Person fetch
+
+  Background:
+    * def domainName = java.lang.System.getenv('DOMAIN_NAME')
+    * def basePath = java.lang.System.getenv('BASE_PATH')
+    * def CRISTIN_BASE =  'https://' + domainName +'/' + basePath
+    * def illegalIdentifier = 'illegalIdentifier'
+    * def nonExistingPersonId = '999999'
+    * def validIdentifier = '738'
+    * def PROBLEM_JSON_MEDIA_TYPE = 'application/problem+json'
+    Given url CRISTIN_BASE
+    * print 'Current base url: ' + CRISTIN_BASE
+
+  Scenario Outline: Unsupported method <METHOD> returns 403
+    Given path '/person/'
+    When method <METHOD>
+    Then status 403
+
+    Examples:
+      | METHOD  |
+      | HEAD    |
+      | PUT     |
+      | DELETE  |
+      | CONNECT |
+
+  Scenario: Unsupported method TRACE returns 405
+    Given path '/person/'
+    When method TRACE
+    Then status 405
```

Comment:
Can you check the Allowed header that is returned?

Reply:
The 405 response does not contain 'Access-Control-Allow-Headers' :-(